pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "switchbot_api"
-version = "2.7.0"
+version = "2.7.1"
```

As we are adding features

```suggestion
version = "2.8.0"
```